charts/incubator/blocky/templates/common.yaml

```diff
@@ -7,12 +7,14 @@
 {{- define "blocky.configmap.mount" -}}
 enabled: true
 type: custom
-mountPath: /app/config/tc-config.yaml
-subPath: tc-config.yaml
+mountPath: /app/config
 readOnly: true
 volumeSpec:
   configMap:
     name: '{{ printf "%s-config" (include "tc.common.names.fullname" .) }}'
+    items:
+      - key: tc-config.yaml
+        path: tc-config.yaml
```

Comment:
With this fix merged: https://github.com/0xERR0R/blocky/pull/656

We should be able to use this type of mounting without problems